Userland/Libraries/LibWeb/Layout/Viewport.h

```diff
@@ -19,12 +19,26 @@ class Viewport final : public BlockContainer {
     explicit Viewport(DOM::Document&, NonnullRefPtr<CSS::StyleProperties>);
     virtual ~Viewport() override;
 
+    struct TextPosition {
+        DOM::Text& dom_node;
```

Comment:
I know, it's probably safe to assume that DOM nodes will live for at least as long as layout tree, but let's always visit GC-allocated members :)